src/azul/plugins/repository/dss/__init__.py

```diff
@@ -366,6 +367,15 @@ def drs_uri(self, drs_path: str) -> str:
         netloc = config.drs_domain or config.api_lambda_domain('service')
         return f'drs://{netloc}/{drs_path}'
 
+    def direct_file_url(self, file_uuid: str, file_version: Optional[str]) -> Optional[str]:
+        file_url = furl(config.dss_endpoint)
+        file_url.path.add('files').add(file_uuid)
+        # Order in which these are added matters for tests
+        if file_version is not None:
+            file_url.query.add(dict(version=file_version))
```

Comment:
There must be a more convenient way than this `add(dict`

Reply:
No longer applicable, but `file_url.query.params['version'] = file_version` would also have worked.

Reply:
```python
>>> u = furl('http://foo.com')
>>> u.args['foo'] = 'bar'
>>> u
furl('http://foo.com?foo=bar')
```

is the idiomatic approach.